var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -226,7 +226,8 @@ class Fftw(FftwBase):
     provides('fftw-api@2', when='@2.1.5')
     provides('fftw-api@3', when='@3:')
 
-    patch('pfft-3.3.5.patch', when="@3.3.5:+pfft_patches", level=0)
+    patch('pfft-3.3.9.patch', when="@3.3.9:+pfft_patches", level=0)
```

Comment:
Did you create this patch yourself (either in full or modified from `pfft-3.3.5.patch`)?

Reply:
I have modified the pfft-3.3.5.patch so that it can be applied to FFTW 3.3.9.

Reply:
I have been looking at #2255 and it seems to me that the patches here are required by a dependent (PFFT). Would it be a good idea to move, in this or another PR, the patch to `pfft` and remove the variant here?

Reply:
> Would it be a good idea to move, in this or another PR

Yes but IMO another PR is best

> I have modified the pfft-3.3.5.patch so that it can be applied to FFTW 3.3.9.

Thanks for that!